pennylane/ops/qubit/matrix_ops.py

```diff
@@ -332,3 +334,197 @@ def _controlled(self, control):
 
     def label(self, decimals=None, base_label=None, cache=None):
         return super().label(decimals=decimals, base_label=base_label or "U", cache=cache)
+
+
+class BlockEncode(Operation):
+    r"""BlockEncode(a, wires)
+    Construct a unitary :math:`U(A)` such that an arbitrary matrix :math:`A`
+    is encoded in the top-left block.
+
+    .. note::
```

Comment:
I prefer if the note goes at the end, not beginning.

Reply:
Moved the note to the end, after the example.